web-components/src/components/tooltip/Tooltip.ts

```diff
@@ -60,14 +60,22 @@ export namespace Tooltip {
       this.handleKeyDown(e);
     };
 
+    private _wheelListener = (e: Event) => {
+      if (this.opened) {
+        this.notifyTooltipDestroy();
+      }
+    };
+
     connectedCallback() {
       super.connectedCallback();
       document.addEventListener("keydown", this._keyDownListener);
+      window.addEventListener("wheel", this._wheelListener, { passive: true });
```

Comment:
![Screenshot 2024-09-03 at 11 04 02](https://github.com/user-attachments/assets/1f767b2c-5d1c-41a7-8b36-3968ee970316)

I'm a little concerned about using this event listener, due to the sheer volume of events generated whenever the user scrolls. However, the logic in the listener does guard the expensive call from being made more than once, as shown with the logs above. 

The reason I'm using wheel events rather than scroll events is because the scroll events are captured by the actual element which is being scrolled. In many cases, this is often a container div rather than the window itself. As such, the tooltip may be within one or more other scrollable containers, none of which can be efficiently obtained from the tooltip itself, without doing an exhaustive search of the DOM, so far as I'm aware. Additionally, because these elements have `overflow: something` in their CSS, even setting `window.addEventListener("scroll", ()=> {...}, true)` won't get called.  

Wheel events are fired to each element, regardless of where they are in the DOM.

The other main use cases in which scroll events are generated without wheel events are largely irrelevant to this specific code change:
- keyboard navigation: in this case, on the first click of the arrow keys, the tooltip disappears. This is also the behaviour if the key is held down.
- Using the mouse on the scroll bar: in this case, the mouse cannot be hovering over an element with a tooltip
- scrolling with JavaScript: haven't tested this use case, but in general terms as soon as the scroll ends, the tooltip will disappear, so it isn't regression in any case.

https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event